package.json

```diff
@@ -27,7 +27,7 @@
     "build-installer": "babel -s --extensions \".ts\" --out-dir packages/playwright-core/lib/utils/ packages/playwright-core/src/utils",
     "doc": "node utils/doclint/cli.js",
     "lint": "npm run eslint && npm run tsc && npm run doc && npm run check-deps && node utils/generate_channels.js && node utils/generate_types/ --check-clean && npm run lint-tests && npm run test-types && npm run lint-packages",
-    "lint-packages": "node utils/prepare_packages.js --check-clean",
+    "lint-packages": "node utils/workspace.js --ensure-consistent",
```

Comment:
```suggestion
    "lint-packages": "node utils/workspace.js --lint",
```
wdyt?

Reply:
i didn't like `lint` in the first place since "linting" is a non-necessary step and could be omitted. Whereas in our case, we need to make workspace consistent before publishing scripts. 

But I could be convinced otherwise!